_data/staff.yml

```
@@ -47,23 +54,33 @@ moderator:
   color: "#f20"
   priority: 200
   members:
-    - name: "AnteloCAZO"
+    - name: "Antel"
       image: "antel.png"
       media:
         discord: "anteloCAZO#7099"
-    - name: "EstebanMz"
-      image: "esteban.jpg"
+    - name: "ChoquePumper"
+      image: "choque.png"
+    - name: "Macacosky"
+      image: "macacosky.png"
       media:
-        discord: "EstebanMz#4398"
+        discord: "Macacosky#5337"
     - name: "Michel"
       image: "michel.png"
-    - name: "WaluigiMB"
-      image: "walele.png"
-    - name: "ChoquePumper"
-      image: "choque.png"
+      media:
+        discord: "Michel#5963#
```

```suggestion
        discord: "Michel#5963#"
```

Missing double quotes here.